File: server/src/main/java/com/fluffy/exam/ui/request/CreateExamWebRequest.java
package com.fluffy.exam.ui.request;

import com.fluffy.exam.application.request.CreateExamRequest;
import com.fluffy.global.web.Accessor;

public record CreateExamWebRequest(String title) {

    public CreateExamRequest toAppRequest(Accessor accessor) {
        return new CreateExamRequest(title, accessor);
    }
}

File: server/src/main/kotlin/com/fluffy/exam/ui/request/CreateExamWebRequest.kt
package com.fluffy.exam.ui.request

import com.fluffy.exam.application.request.CreateExamRequest
import com.fluffy.global.web.Accessor

data class CreateExamWebRequest(val title: String) {

    fun toAppRequest(accessor: Accessor): CreateExamRequest {
        return CreateExamRequest(
            title = title,
            accessor = accessor
        )
    }
}
